packages/jest/src/generators/init/init.ts

```diff
@@ -68,6 +68,21 @@ function updateDependencies(tree: Tree, options: NormalizedSchema) {
 
   if (options.babelJest) {
     devDeps['babel-jest'] = babelJestVersion;
+    devDeps['@nrwl/js'] = nxVersion; // TODO(caleb): switch to nrwl/js
+    // TODO(caleb): is there where I need to set up babel? feels like this should be generic somewhere
+    devDeps['@babel/core'] = '^7.15.0';
+    devDeps['@babel/preset-env'] = '^7.15.0';
+    devDeps['@babel/plugin-proposal-class-properties'] = '^7.14.5';
+    devDeps['@babel/plugin-proposal-decorators'] = '^7.14.5';
+    devDeps['@babel/plugin-transform-regenerator'] = '^7.14.5';
+    devDeps['@babel/plugin-transform-runtime'] = '^7.15.0';
+    devDeps['@babel/runtime'] = '^7.14.8';
+    devDeps['@babel/preset-typescript'] = '^7.15.0';
+    devDeps['babel-loader'] = '^8.2.2';
+    devDeps['babel-plugin-const-enum'] = '^1.0.1';
+    devDeps['babel-plugin-macros'] = '^2.8.0';
+    devDeps['babel-plugin-transform-async-to-promises'] = '^0.8.15';
+    devDeps['babel-plugin-transform-typescript-metadata'] = '^0.3.1';
```

Comment:
These are a lot 😦 

And I agree with your comment, this should be a function somewhere. If not, then at least these version numbers should be in a constant file; similar to `babelJestVersion`

Reply:
moved them out into a const 👍